falcon/routing/util.py

```diff
@@ -140,6 +140,11 @@ def map_http_methods(resource):
             should define ``on_get(self, req, resp)`` and
             ``on_post(self, req, resp)``.
 
+        suffix: A string used to look for alternate methods to the base HTTP methods.
```

Comment:
Should we specify type like in the docstring earlier in this PR: ``suffix (str): ...``?

Reply:
well the reason I didn't is because I was just trying to match the previous argument in the docstring, the one `resource` above it doesn't have a type with out.